src/styles/partials/template/_sidebar.scss

```diff
@@ -18,7 +18,15 @@
   p {
     font-family: $default-heading-font;
     font-size: $smallest-heading-font-size;
-    margin: 0 0 $default-margin 0;
+  }
+
+  .dg_sidebar__content {
```

Comment:
@theilman7 have you adjusted this markup anywhere?

Reply:
@martypowell Pushed up some corrections and staged on the components page in beta.